cmd/wait-for-github/ci_test.go

```diff
@@ -75,7 +75,7 @@ func TestHandleCIStatus(t *testing.T) {
 		tt := tt
 
 		t.Run(tt.name, func(t *testing.T) {
-			output := handleCIStatus(tt.status, 1)
+			output := handleCIStatus(tt.status, 1, "")
```

Comment:
maybe we could test that the given URL is somewhere in the output?

Reply:
done